src/interfaces/store-interfaces/state-interfaces/registration-state-interface.ts

```diff
@@ -7,19 +7,12 @@ export interface RegistrationNameRequestIF {
   nrNumber: string
 }
 
-export const EmptyRegistrationNameRequest: RegistrationNameRequestIF = {
```

Comment:
No longer needed in state (store).

RegistrationNameRequestIF is now only used to declare the filing object sent to the API.